shared/actions/tracker.js

```diff
@@ -51,14 +51,13 @@ export function stopTimer (): Action {
 }
 
 export function registerTrackerChangeListener (): TrackerActionCreator {
-  return dispatch => {
+  return (dispatch, getState) => {
     const params = {
       'keybase.1.NotifyTracking.trackingChanged': ({username}) => {
-        dispatch({
-          type: Constants.userUpdated,
-          payload: {},
-        })
-        dispatch(getProfile(username))
+        const trackerState = getState().tracker.trackers[username]
+        if (trackerState && trackerState.type === 'tracker' && !trackerState.closed) {
```

Comment:
They share the same store state so if I'm looking at a profile and this
comes in then I'll update and the tacking state will be correct. Is there
another mechanism we use to update?

On Friday, August 5, 2016, Marco Munizaga <notifications@github.com> wrote:

> In shared/actions/tracker.js
> <https://github.com/keybase/client/pull/3759#discussion_r73754627>:
>
> >      const params = {
> >        'keybase.1.NotifyTracking.trackingChanged': ({username}) => {
> > -        dispatch({
> > -          type: Constants.userUpdated,
> > -          payload: {},
> > -        })
> > -        dispatch(getProfile(username))
> > +        const trackerState = getState().tracker.trackers[username]
> > +        if (trackerState && trackerState.type === 'tracker' && !trackerState.closed) {
>
> those don't go in this code path
>
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/keybase/client/pull/3759/files/cd8fb4e1984b1fb75998561a810ddc62a655a604#r73754627>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAfhU7Z50uLSwZzRqD7iDUX8a5fdCkjJks5qc51WgaJpZM4JeB9_>
> .
>